tests/e2e_tests/e2e_test_config.yml

```diff
@@ -14,7 +14,17 @@ LANGUAGES:
     name: polski
     flag: pl
 
-
 #TODO this should not be necessary argument
 ROUTE_OVERWRITES:
   /static/map.js: https://cdn.jsdelivr.net/npm/@problematy/goodmap@0.1.6
+
+SMTP:
+  PORT: 465
+  SERVER: "smtp.gmail.com"
+  ADDRESS: "my@gmail.com"
+  PASSWORD: "PA$$WORD"
+  DEFAULT_SENDER: "sender@gmail.com"
+  ADMIN_EMAIL: "help.desk@gmail.com"
```

Comment:
After adding `PLUGINS`, this should no longer be necessary.

Reply:
Yup, I thought about this, but later I got reflection, that we are testing if app starts at all. So should we test if only base part of an app works, or should we test whole app. If we test only a part of it, then how we are sure if someone who will want to use reporting location won't end up with broken app

Reply:
Oh. I totally agree that we should test whole app. :)
Yet - there is no e2e test yet (which should be here).

Reply:
Created #30 to track that.

And to be honest - I don't have good idea how to do this. (I have ideas, but would not say they are good). So please enjoy taking part in #30 discussion.

Reply:
And summing it up. I'd remove this config and plugin (which are not tested now) and move it to this second issue.
Which should be taken right away.

Reply:
Done

Reply:
It seems that it's still here. Did you forget do `commit push`?